### DISCUSSION TITLE
NMF associate spot to factor

### DISCUSSION THREAD

**Discussion by Nasrine26** [BayraktarLab/cell2location#138] [Q&A]
Hello,

I am trying to use the output of NMF for further downstream analysis and am wondering how do I know which spots are associated to which factor? 

Many thanks

**Comment by vitkl** [comment]
Factor values across spots are continuous - no not discrete labels.

The values are stored in `adata.uns[f"mod_coloc_n_fact{n_fact}"]` in a similar output format main cell2location results.
